pkg/compose/build.go

```diff
@@ -81,6 +81,12 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opti
 				Attrs: map[string]string{"ref": image},
 			})
 		}
+		buildOptions.Exports = []bclient.ExportEntry{{
```

Comment:
Nit: it'd be a bit clearer if it was something like

```go
if len(buildOptions.Platforms) == 0 {
   // docker load
} else { 
  // image out
}
```

Mostly to make it clear that they're mutually exclusive / an intentional override.